t/full-stack.t

```diff
@@ -220,7 +225,7 @@ OpenQA::Test::FullstackUtils::client_call(
     qr|\Qtest_url => [{ 1 => "/tests/2\E|,
     'client returned new test_url'
 );
-#]} restore syntax highlighting
+#]| restore syntax highlighting
```

Comment:
why do you need to change this?

Reply:
Because it didn't work. Maybe some other changes broke it or it never (fully) worked.